### ISSUE TITLE
implicit declaration of function 'SystemClock_Config';

### ISSUE THREAD

**Issue by yparitcher** [solokeys/solo#128]
when building solo i get an error
```
bootloader/main.c:122:5: warning: implicit declaration of function 'SystemClock_Config'; did you mean 'SysTick_Config'? [-Wimplicit-function-declaration]
     SystemClock_Config();
     ^~~~~~~~~~~~~~~~~~
     SysTick_Config
```
https://github.com/solokeys/solo/blob/1ff00895b1649c37a893ca49b5f666c9966cd81f/targets/stm32l432/bootloader/main.c#L122

is missing its declaration which was removed from
https://github.com/solokeys/solo/blob/master/targets/stm32l432/src/init.h
in https://github.com/solokeys/solo/commit/ff0d42c8d54574b0bee35b4812528f1b0802e4d7

please fix:

i am not sure what is needed as this looks related to the nfc startup.

**Comment by yparitcher** [comment]
this can also be seen in the travis builds

**Comment by yparitcher** [comment]
fixed in 50bfbc1eff01b98fa1aa242adc6b68fd17ebe3fb
